org.bndtools.builder/src/org/bndtools/builder/impl/CnfWatcher.java

```diff
@@ -37,9 +37,11 @@
 	@Reference
 	BndtoolsBuilder					builder;
 
-	// Bnd workspace, not Eclipse workspace.
 	@Reference
-	Workspace						workspace;
+	IWorkspace						eclipseWorkspace;
```

Comment:
I like these var names to clearly distinguish between the 2 workspaces!

Reply:
:+1: I got confused one too many times...